src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioMetadataReference.Snapshot.cs

```diff
@@ -110,7 +110,7 @@ protected override PortableExecutableReference WithPropertiesImpl(MetadataRefere
         private string GetDebuggerDisplay()
             => "Metadata File: " + FilePath;
 
-        public IReadOnlyList<ITemporaryStreamStorageInternal> GetStorages()
-            => _provider.GetStorages(this.FilePath, _timestamp.Value);
+        public IReadOnlyList<TemporaryStorageIdentifier> GetStorageIdentifiers()
+            => _provider.GetStorageIdentifiers(this.FilePath, _timestamp.Value);
```

Comment:
the old name was just super confusing.  what does 'GetStorages' even mean?  Also, what it returned was also weird.  It returned the old object htat you could read/write to (though nothing ever read/wrote to it).  So what was it used for?  Well, callers would cast this type to another type to try to get the MMF info to stream to OOP.

All this gets simpler by havnig a real type to represetnt that data, and having tehse helpers just return instances of those types alone.  Not the actual storage object.